nodejs/aws-infra/experimental/ec2/securityGroup.ts

```diff
@@ -37,6 +37,16 @@ export class SecurityGroup extends pulumi.ComponentResource {
         this.registerOutputs();
     }
 
+    public static fromExistingId(
+        name: string, id: pulumi.Input<string>,
+        args: SecurityGroupArgs = {}, opts: pulumi.ComponentResourceOptions = {}) {
+
+        return new SecurityGroup(name, {
+            ...args,
+            instance: aws.ec2.SecurityGroup.get(name, id, {}, opts),
```

Comment:
I honestly don't know at this point - but is it safe to pass `ComponentResourceOptions` to a `.get(...)`.  

/cc @pgavlin

Reply:
I believe so, yes.